js/src/main/scala/crystal/react/reuse/Reuse.scala

```diff
@@ -33,22 +33,26 @@ import scala.reflect.ClassTag
 trait Reuse[+A] {
   type B
 
-  val get: () => A
+  lazy val value: A = getValue()
 
-  lazy val value: A = get()
+  protected[reuse] val getValue: () => A
```

Comment:
Why was this necessary?

Reply:
Not necessary. But we discussed it. The renaming would have been enough. But, it seems like `value` will be the one used, not `getValue`.

Reply:
The name `get` conflicted with `Reuse,get` wrt the extension methods.

Reply:
Oh ok sorry. My brain had trouble processing the fact that they switched positions 👍